src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/cloudconfigs/fdm/salesforce-cloud-config/_jcr_content/swaggerFile

```diff
@@ -170,7 +170,163 @@
 					}
 				}
 			}
-		}
+		},
+        "/services/data/v32.0/sobjects/Account/{id}": {
+            "get": {
+                "summary": "Fetch Account Information From SDFC",
```

Comment:
Is it SDFC or SFDC? Can we correct it throughout.

Reply:
changed all places to SFDC